Saint-Nec-Engine/Saint-Nec-Engine-Lib/src/architecture/Time.hpp

```diff
@@ -55,6 +50,22 @@ class Time
 	 */
 	static float getTimeSinceStart();
 private:
+
+	/**
+	 * Default constructor.
+	 */
+	Time();
+
+	/**
+	 * Destructor.
+	 */
+	~Time();
```

Comment:
Not sure Destructor might be private, constructor is enough to ensure singleton.
However [here](https://stackoverflow.com/questions/631783/what-is-the-use-of-having-destructor-as-private) we can see that it could be good if something else need to manage the destruction (I don't think it's our case)